justfile

```diff
@@ -29,3 +30,14 @@ down:
 # Pull all docker images
 pull:
     {{ DC }} pull
+
+@test-int test_path="":

```

Comment:
This should probably depend on the `build` recipe.

Reply:
I don't want to pollute the output of the integration testing with the build recipes "not quiet" version of the run, which is why I'm running the dc up part manually within it instead of having it depend on the build recipe.

Reply:
Got it. I worry that the obfuscation of the output of these docker steps might make it hard to troubleshoot issues with the integration tests.

Reply:
You are correct, it certainly made diagnosing the issue I was having just now pretty annoying, but I still would prefer to have the test output be the only output for this recipe because it's already a bunch of data to begin with.